packages/flutter/test/widgets/async_test.dart

```diff
@@ -180,6 +273,19 @@ void main() {
       await eventFiring(tester);
       expect(find.text('AsyncSnapshot(ConnectionState.done, 4, null)'), findsOneWidget);
     });
+    testWidgets('runs the builder using given initial data', (WidgetTester tester) async {
+      final String initial = 'A';
```

Comment:
Please use a different value than `'A'`. That value is associated with stream A or future A in other test cases.